src/GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp/local.settings.json

```diff
@@ -3,10 +3,11 @@
   "Values": {
     "AzureWebJobsStorage": "UseDevelopmentStorage=true;DevelopmentStorageProxyUri=http://data-storage",
     "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated",
-    "ReleaseVersionPublishedQueueName": "release-version-published-queue",
+    "PublicationChangedQueueName": "publication-changed-queue",
```

Comment:
Related PR to add this app setting and create the queue - https://github.com/dfe-analytical-services/explore-education-statistics/pull/5761